### PR TITLE
add doc for layers/normalization/BatchNormalization

### PR DESCRIPTION
**Pull request by MoyanZitto**
The BN layer use ```set_weights``` to load weights provided by user, and in ```set_weights```:
```python
params = self.trainable_weights + self.non_trainable_weights
```
In BN, the trainable_weights are Gamma and Beta, non_trainable_weights are Mean and Std, So it important to point out that the weights list must be organized as [Gamma, Beta, Mean, Std] so that it could work properly. Cause by defualt, people may want to organize their weight list as [Mean, Std, Gamma, Beta], this is just the computation order of BN.

The wrong-order list won't get any warnings or errors cause the shape of these 4 parameters are exactly same.

I think this doc will be helpful.

### CODE REVIEW

File: keras/layers/normalization.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ class BatchNormalization(Layer):
         weights: Initialization weights.
             List of 2 Numpy arrays, with shapes:
             `[(input_shape,), (input_shape,)]`
+			Note that the order of this list is [Gamma, Beta, mean, std]
```
Comment (fchollet) [inline keras/layers/normalization.py]: This is getting the PEP8 test to fail. You don't need this crazy indentation. You don't need to capitalize the variable names.

Comment (MoyanZitto) [inline keras/layers/normalization.py]: Really sorry for the wrong pushs. I didn't see any wrong indentation in my notepad++ editor, anyway I'll check and push again.

BTW, I've translated Keras documents into Chinese. I think this Chines version documents would be helpful for those who use Keras. The homepage of this document is http://keras-cn.readthedocs.io/en/latest/ , you may want to add this link to keras.io. Thank you~ :）
